pom.xml

```diff
@@ -23,6 +23,9 @@
     <jetty.version>9.4.30.v20200611</jetty.version>
     <slf4j.version>1.7.30</slf4j.version>
     <logback.version>1.2.3</logback.version>
+    <!-- Make sure micrometer and prometheus versions are in sync  -->
+    <micrometer.version>1.6.5</micrometer.version>
+    <prometheus.version>0.9.0</prometheus.version>
```

Comment:
Why not add `prometheus` to the dep. management section below where we already have `micrometer`?